zephyr/module.yml

```diff
@@ -5,47 +5,11 @@ build:
   kconfig-ext: True
 blobs:
 
-    - path: wifi_fw_bins/default/nrf70.bin
```

Comment:
remove comment from line #1

Reply:
wait, is this still autogenerated?

Reply:
actually, I'm still using the script. Fixed it now!